### ISSUE TITLE
"this TensorFlow binary was not compiled to use" ...  Aborted (core dumped)

### ISSUE THREAD

**Issue by ruochongli** [tensorflow/nmt#448]
Hi, I met this problem when I tried to run the following code on GPU:

python -m nmt.nmt \
    --src=en --tgt=zh \
    --vocab_prefix=nmt/wmt17/vocab.bpe.32000  \
    --train_prefix=nmt/wmt17/train.tok.clean.bpe.32000 \
    --dev_prefix=nmt/wmt17/newsdev2017.tok.bpe.32000 \
    --test_prefix=/wmt17/newstest2017.tok.bpe.32000 \
    --out_dir=nmt/nmt_model \
    --num_train_steps=12000 \
    --steps_per_stats=100 \
    --num_layers=2 \
    --num_units=128 \
    --dropout=0.2 \
    --metrics=bleu

Also, running the following code will give the same error:

python -m nmt.nmt \
    --src=en --tgt=zh \
    --hparams_path=nmt/standard_hparams/wmt16.json \
    --out_dir=nmt/deen_gnmt \
    --vocab_prefix=nmt/wmt17/vocab.bpe.32000 \
    --train_prefix=nmt/wmt17/train.tok.clean.bpe.32000 \
    --dev_prefix=nmt/wmt17/newsdev2017.tok.bpe.32000 \
    --test_prefix=nmt/wmt17/newstest2017.tok.bpe.32000

The picture of the error: 
<img width="728" alt="屏幕快照 2019-09-30 下午8 58 56" src="https://user-images.githubusercontent.com/55910562/65890254-29c7a600-e3d5-11e9-9274-81eba48ffe47.png">

Can anyone help? Is this a compiling problem? How can I tackle with it? Or is this a problem caused by my dataset?

Thank you very much.

**Comment by nashid** [comment]
I also had the same issue.
